i18n/resources/en/login.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   signInFRA: 'Sign in with FRA',
 
   resetPassword: 'Reset password',
+  changePassword: 'Change password',
```

Comment:
add to githib notes missing translations

Reply:
changePassword is not missing, resetPassword is and it's already in the missing translations list